packages/ember-data/lib/adapters/rest_adapter.js

```diff
@@ -549,6 +549,18 @@ export default Adapter.extend({
     return url;
   },
 
+
+  
+  buildURLForHasMany: function(relationshipName, type, record) {
```

Comment:
I am not sure we want provide a default implementation, seems more like a fallback option?